include/onnxruntime/core/graph/graph.h

```diff
@@ -1391,6 +1438,20 @@ class Graph {
   Node& AddNode(const ONNX_NAMESPACE::NodeProto& node_proto,
                 const ArgNameToTypeMap& name_to_type);
 
+  /** Helper that converts and adds constant node proto to an initializer in the graph.
+      The function requires that all of the nodes names in the constant_node_proto has already been specialized.
```

Comment:
Maybe "requires that the name of the output-tensor of constant_node_proto must be unique in the graph" or something like that? Also, I guess the model_path is to handle externally stored tensors?